x/gov/client/cli/tx.go

```diff
@@ -132,7 +132,7 @@ func GetCmdDeposit(queryRoute string, cdc *codec.Codec) *cobra.Command {
 		Long: strings.TrimSpace(`
 Submit a deposit for an acive proposal. You can find the proposal-id by running gaiacli query gov proposals:
 
-$ gaiacli tx gov deposit 1 10stake --from mykey
```

Comment:
anything in `x/` referencing gaia should probably be removed entirely or moved into the gaia repo. (aka redefine `cobra.Command.Long` in gaia) - This is fine temporarily I guess.

Reply:
There are more than fifty `gaiacli` in `x/`. Should them be modified in another PR ?